.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1.0
+
+workflows:
+  node-tests:
+    jobs:
+      - node/test:
+          version: 14.15.4
```

Comment:
that's one simple job! too bad it does not support publishing the test reports out of the box